apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PageLinkBuilder.java

```diff
@@ -179,41 +155,27 @@ public void addPageLinks(Bundle toBundle, int numTotalResults) {
    */
   private String createPageLink(int startIndex) {
     StringBuilder b = new StringBuilder();
+
```

Comment:
Now that we are really improving this method. I'm wondering if we could use UriBuilder instead of StringBuilder here.

Reply:
That sounds interesting. Will check that out :-)

Reply:
@RickHawesUSDS I changed it to use the URIBuilder. This worked out well. I'll update how the links now look in the change details.  TY!